distutils/sysconfig.py

```diff
@@ -120,8 +120,11 @@ def get_python_inc(plat_specific=0, prefix=None):
     """
     default_prefix = BASE_EXEC_PREFIX if plat_specific else BASE_PREFIX
     resolved_prefix = prefix if prefix is not None else default_prefix
+    # MinGW imitates posix like layout, but os.name != posix
+    os_name = os.name if not sysconfig.get_platform().startswith("mingw") else "posix"
```

Comment:
Is there a reason this check doesn't use `is_mingw()`?

Reply:
This module didn't import `distutils.sysconfig` before and I thought that was done for a reason. If that's not the case, I'll use `is_mingw()` function.

Reply:
I don't see any protections against importing it (e.g. a comment indicating not to import other modules), so I'd say it's safe. Let's re-use it. Maybe something like this:

```diff
diff --git a/distutils/sysconfig.py b/distutils/sysconfig.py
index 35e5289..0fbb2e1 100644
--- a/distutils/sysconfig.py
+++ b/distutils/sysconfig.py
@@ -18,6 +18,7 @@ import pathlib
 from .errors import DistutilsPlatformError
 from . import py39compat
 from ._functools import pass_none
+from .util import is_mingw
 
 IS_PYPY = '__pypy__' in sys.builtin_module_names
 
@@ -121,7 +122,7 @@ def get_python_inc(plat_specific=0, prefix=None):
     default_prefix = BASE_EXEC_PREFIX if plat_specific else BASE_PREFIX
     resolved_prefix = prefix if prefix is not None else default_prefix
     # MinGW imitates posix like layout, but os.name != posix
-    os_name = os.name if not sysconfig.get_platform().startswith("mingw") else "posix"
+    os_name = "posix" if is_mingw() else os.name
     try:
         getter = globals()[f'_get_python_inc_{os_name}']
     except KeyError:
```